.env.example

```diff
@@ -32,10 +32,10 @@ MAIL_ENCRYPTION=null
 MAIL_FROM_ADDRESS=null
 MAIL_FROM_NAME="${APP_NAME}"
 
-CAS_HOSTNAME=login.gatech.edu
-CAS_REAL_HOSTS=login.gatech.edu
+CAS_HOSTNAME=cas-test.gatech.edu
+CAS_REAL_HOSTS=cas-test.gatech.edu
 CAS_SESSION_NAME=session
-CAS_LOGOUT_URL=login.gatech.edu/cas/logout
```

Comment:
Don't change the example file since in general login.gatech.edu is correct, just not for the next two weeks or however long it is.

Reply:
To clarify, these changes will need to be made in the actual `.env` file on dc00. `.env.example` is just an example.